modules/ROOT/pages/manage-monitor-applications.adoc

```diff
@@ -3,15 +3,15 @@
 Applications and API gateways deployed to Runtime Fabric include native support for Anypoint Monitoring to allow viewing metrics within Anypoint Platform.
 
 [NOTE]
-Monitoring solutions other than Anypoint Monitoring are not supported.
+Runtime Fabric does not support monitoring solutions other than Anypoint Monitoring.
```

Comment:
same wording as above - "provide support"

Reply:
yes i like the provide support